code/modules/projectiles/projectile/bullets/cannonball.dm

```diff
@@ -7,7 +7,7 @@
 	projectile_piercing = ALL
 	dismemberment = 0
 	paralyze = 5 SECONDS
-	stutter = 10 SECONDS
```

Comment:
Since `stutter` wasn't actually a duration (1 stutter = 2 seconds), cannonballs used to stutter you for 100 (200 seconds)